src/execute_linter.cc

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include <cstdio>
+#include <iostream>
+#include <memory>
+#include <vector>
+#include <string>
+#include <fstream>
+#include <streambuf>
+
+#include "absl/strings/string_view.h"
+#include "src/lint_errors.h"
+#include "src/linter.h"
+#include "zetasql/base/status.h"
+#include "zetasql/base/status_macros.h"
+
+namespace zetasql::linter {
+
+    // It runs all specified checks
+    // "LinterOptions" parameter will be added in future
+    absl::Status run_checks(absl::string_view sql) {
+        LinterResult result, output;
+        ZETASQL_RETURN_IF_ERROR(CheckLineLength(sql, &output));
+        result.Add(output);
+        output.clear();
+        ZETASQL_RETURN_IF_ERROR(CheckParserSucceeds(sql, &output));
+        result.Add(output);
+        output.clear();
+        ZETASQL_RETURN_IF_ERROR(CheckSemicolon(sql, &output));
+        result.Add(output);
+        output.clear();
+        ZETASQL_RETURN_IF_ERROR(CheckUppercaseKeywords(sql, &output));
+        result.Add(output);
+        output.clear();
+        ZETASQL_RETURN_IF_ERROR(CheckCommentType(sql, &output));
+        result.Add(output);
+        output.clear();
+        ZETASQL_RETURN_IF_ERROR(CheckAliasKeyword(sql, &output));
+        result.Add(output);
+        std::cout << "Linter completed running." << std::endl;
+        result.PrintResult();
+        return absl::OkStatus();
+    }
```

Comment:
I wonder how can we test that this function includes **all** implemented checks?

Reply:
I am planning to hold all checks in a structure, then we can see if all of them is here.